openapi.yaml

```diff
@@ -2165,93 +2177,752 @@ paths:
+      operationId: updateAssessmentResults
+      parameters:
+        - name: assessmentResultId
+          in: path
+          description: ID of catalog to return
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Catalog object to be updated
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/OSCALAssessmentResultsUpdateExample"
+        required: true
+      responses:
+        400:
+          description: Invalid ID supplied
+          content: {}
+        404:
+          description: Catalog not found
+          content: {}
+        405:
+          description: Validation exception
+          content: {}
+        409:
+          description: Conflict with current state
+      security:
+        - oscal_auth:
+            - write:catalogs
+            - read:catalogs
+      x-codegen-request-body-name: body
+    put:
+      tags:
+        - OSCAL System Assessment Results
+      summary: Replaces an existing OSCAL System Assessment Result
+      operationId: replaceAssessmentResults
+      parameters:
+        - name: assessmentResultId
+          in: path
+          description: ID of catalog to replace.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Catalog object to be replaced.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/OSCALAssessmentResults"
+        required: true
+      responses:
+        200:
+          description: Updated catalog
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OSCALAssessmentResults"
+        400:
+          description: Bad Request
+        404:
+          description: Catalog not found
+        409:
+          description: Conflict with current state
+        415:
+          description: Unsupported media type
+      security:
+        - oscal_auth:
+            - write:catalogs
+            - read:catalogs
+      x-codegen-request-body-name: body
+    delete:
+      tags:
+        - OSCAL System Assessment Results
+      summary: Deletes an OSCAL System Assessment Result
+      operationId: deleteAssessmentResults
+      parameters:
+        - name: api_key
+          in: header
+          schema:
+            type: string
+        - name: assessmentResultId
+          in: path
+          description: Catalog ID to delete
+          required: true
+          schema:
+            type: string
+      responses:
+        400:
+          description: Invalid ID supplied
+          content: {}
+        404:
+          description: Catalog not found
+          content: {}
+      security:
+        - oscal_auth:
+            - write:catalogs
+            - read:catalogs
+  /system-assesment-plan/search:
+    get:
+      tags:
+        - OSCAL System Assessment Results
```

Comment:
```suggestion
        - OSCAL System Security Plan
```

Reply:
Why should we change this to Security Plan when its a System Assessment Plan?

Reply:
I guess it should actually be `OSCAL System Assessment Plans`?

Reply:
We're searching for a SAR here, so it makes sense for this to remain as `OSCAL System Assessment Results`.